docnado/docs/documents/links.md

```diff
@@ -19,7 +19,7 @@ For example:
 This is a sentence with a link to the [Google Homepage](http://www.google.co.uk "Google Search Homepage") in it.
 ```
 
-This is a sentence with a link to the [Google Homepage](http://www.google.co.uk "Google Search Homepage") in it.
+This is a sentence with a link to the [Google Homepage](http://www.gle.co.uk/ouashd9o.html "Google Search Homepage") in it.
```

Comment:
I'd consider a separate broken link example to avoid confusion in the official docs.

Reply:
This was a mistake left over from testing. I committed it in error.